### ISSUE TITLE
webpack: missing bundles

### ISSUE THREAD

**Issue by zzacharo** [inveniosoftware/invenio-previewer#109]
In this [file](https://github.com/inveniosoftware/invenio-previewer/blob/master/invenio_previewer/assets/js/pdfjs/viewer.js#L36) there is a dependency on `/static/js/pdfjs/pdf.worker.min.js` but the file is not bundled. We should either include it in the `static` file or create a specific bundle for that. Or align [assets/js/pdfjs/viewer.js](https://github.com/inveniosoftware/invenio-previewer/blob/master/invenio_previewer/assets/js/pdfjs/viewer.js#L36) with [static/js/pdfjs/viewer.js](https://github.com/inveniosoftware/invenio-previewer/blob/master/invenio_previewer/static/js/pdfjs/viewer.js#L46).

**Comment by egabancho** [comment]
Is this somehow connected with this error message?

```
The resource from “https://127.0.0.1:5000/static/js/pdfjs/pdf.worker.min.js” was blocked due to MIME type (“text/html”) mismatch (X-Content-Type-Options: nosniff). loremipsum.pdf
```

I'm using Invenio-Previewer `1.1.0` and Firefox `73.0.1`

**Comment by zzacharo** [comment]
@egabancho what I noticed is that the file is required but not placed correctly under `/static`. Your issue seems more like a CSP one. In my case the pdf preview is working properly but still I think the fact that the file is not there needs to be addressed by removing it if is not necessary or fixing it :)

**Comment by ppanero** [comment]
Tl;DR:

- Files are duplicated in assets and statics.
- When in assets, the import works because, afterwards is importing from static.
- Any import in js, should be bundled in webpack and removed from static. This might require to put those files in it's own bundle.
